XMonad/Layout/Minimize.hs

```diff
@@ -112,7 +112,7 @@ instance LayoutModifier Minimize Window where
                 case M.lookup w (W.floating ws) of
                   Nothing -> return $ Just $ Minimize (w:minimized) unfloated
                   Just r -> do
-                    modify (\s -> s { windowset = W.sink w ws})
+                    windows (W.sink w)
```

Comment:
In particular, it's going to re-run the layout. This seems like a bad thing to do from the layout.